plugins/nextcloud/nextcloud_

```diff
@@ -47,15 +50,23 @@ GPLv2
 
 =cut
 
+# shellcheck disable=SC1090
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
-API_PATH="/ocs/v2.php/apps/serverinfo/api/v1/info?format=json"
+if [ "${MUNIN_DEBUG:-0}" = 1 ]; then
+    set -x
+fi
+
+API_PATH="${api_path:-/ocs/v2.php/apps/serverinfo/api/v1/info}?format=json"
 DOMAIN="${0##*nextcloud_}"
+TIMEOUT="${timeout:-2}"
+UPDATES_WARNING="${updates_warning:-1}"
 CLEANDOMAIN="$(clean_fieldname "$DOMAIN")"
 USERNAME="${username:-}"
 PASSWORD="${password:-}"
 
 print_json_data() {
+    # shellcheck disable=SC2039
```

Comment:
You may omit this, if you like, since `shellcheck` is called as a CI test with the arguments `--exclude=SC1090,SC2009,SC2126,SC2230 --shell=dash` (see `t/test.t`). Sadly this is not configurable in a separate file, thus there is no independent way of running `shellcheck` for a separate file withe the arguments used by the CI job.

Reply:
I added a `# shellcheck shell=dash` directive